pycbc/io/live.py

```diff
@@ -105,7 +105,8 @@ def __init__(self, ifos, coinc_results, **kwargs):
             Strain channel names for each detector.
             Will be recorded in the sngl_inspiral table.
         """
-        self.template_id = coinc_results['foreground/%s/template_id' % ifos[0]]
+        self.template_id = \
+                coinc_results['foreground/%s/template_id' % next(iter(ifos))]
```

Comment:
It feels like the better approach here would be to cast ifos to a list before coming into this function. The function documentation says it is expecting a list.

Reply:
Agreed.